From e505f39c2f5c818b061d89cddde2962d349c2147 Mon Sep 17 00:00:00 2001
From: Potnuri Bharat Teja <bharat@chelsio.com>
Date: Tue, 26 Mar 2019 12:37:38 +0530
Subject: [PATCH 01/10] cxgb4: Removing re-occurant logs which provide
 information of non-VLAN Tag on DCB Link

Note: This print is still valid, but may not need in out current use case.

Signed-off-by: Potnuri Bharat Teja <bharat@chelsio.com>
---
 drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
index 3e7414f91bc7..09d4c5f04b86 100644
--- a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
@@ -1076,9 +1076,6 @@ static u16 cxgb_select_queue(struct net_device *dev, struct sk_buff *skb,
 
 		err = vlan_get_tag(skb, &vlan_tci);
 		if (unlikely(err)) {
-			if (net_ratelimit())
-				netdev_warn(dev,
-					    "TX Packet without VLAN Tag on DCB Link\n");
 			txq = 0;
 		} else {
 			txq = (vlan_tci & VLAN_PRIO_MASK) >> VLAN_PRIO_SHIFT;
-- 
2.24.0

